xbmc/events/BaseEvent.cpp

```diff
@@ -29,6 +32,7 @@ CBaseEvent::CBaseEvent(const std::string& identifier, const CVariant& label, con
     m_description(description),
     m_details(),
     m_executionLabel(),
+    m_timestamp(std::chrono::time_point_cast<std::chrono::nanoseconds>(std::chrono::steady_clock::now()).time_since_epoch().count()),
```

Comment:
First, it seems that you looked  not at the most recent commit? Meanwhile I moved the chrono stuff into a static member function, ctors call this function no, so no more code duplication here.

Second, I wanted to avoid chrono data types in the header file. That's why I decided to have a primitive type as member.

Okay?